src/main/java/am/ik/yavi/constraint/CharacterConstraint.java

```diff
@@ -45,4 +45,14 @@
 	protected Predicate<Character> isLessThanOrEqual(Character max) {
 		return x -> x <= max;
 	}
+
+	@Override
+	protected Character zeroValue() {
+		return Character.MIN_VALUE;
```

Comment:
`'0'` is explicit and better to read

Reply:
The difference between `Character.MIN_VALUE` and '0' is that the first represent `U+0000` and the second is just the character

Reply:
Ah, right.
So is that the numeric 'zero value' ??
No tests added for `CharacterConstraintTest`.

Reply:
```java
System.out.println(Character.getNumericValue('0')); // => 0
System.out.println(Character.getNumericValue(Character.MIN_VALUE)); // => -1
```

So, I think `'0'` is correct.

Reply:
Yeah as far as I can see if you cast `Character.MIN_VALUE` to integer you get the numeric value of 0

Reply:
Ok I will change it to '0'

Reply:
Never mind. `Character.MIN_VALUE` is correct.

```java
int a = '0';
System.out.println(a); // => 48
int b = Character.MIN_VALUE;
System.out.println(b); // => 0
```

I think I was confused by the `oneValues` implementation...

```java
	@Override
	protected Character oneValue() {
		return '1';
	}
```

Reply:
Oh... i will change it back to `Character.MIN_VALUE`